sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/implementation/Utility.java

```diff
@@ -129,19 +134,30 @@ static void validateNettyVersions(Consumer<String> logger) {
 
             // Version shouldn't be null as azure-core-http-netty has it as a dependency, but it could have been
             // excluded. Don't include it as a warning for native dependencies as it is optional.
-            if (version != null && !Objects.equals(version.artifactVersion(), nettyTcnativeVersion)) {
-                versionMismatches.add("'io.netty:" + artifact + "' version: " + version.artifactVersion()
-                    + " (expected: " + nettyTcnativeVersion + ")");
+            if (version != null) {
+                versionInformation.add("'io.netty:" + artifact + "' version: " + version.artifactVersion());
             }
         }
 
-        if (!versionMismatches.isEmpty()) {
-            logger.accept("The following Netty dependencies have versions that do not match the versions specified in "
-                + "the azure-core-http-netty pom.xml file. This may result in unexpected behavior. If your application "
-                + "runs without issue this message can be ignored, otherwise please update the Netty dependencies to "
-                + "match the versions specified in the pom.xml file. Versions found in runtime: "
-                + CoreUtils.stringJoin(",", versionMismatches));
+        String versionInformationString = CoreUtils.stringJoin(", ", versionInformation);
+        StringBuilder stringBuilder
+            = new StringBuilder().append("The following is Netty version information that was found on the classpath: ")
+                .append(versionInformationString)
+                .append(". ");
+
+        if (nonNativeNettyVersions.size() > 1) {
+            stringBuilder.append(NETTY_VERSION_MISMATCH_LOG);
         }
+
+        stringBuilder.append("The version of azure-core-http-netty being used was built with Netty version ")
+            .append(nettyVersion)
+            .append(" and Netty Tcnative version ")
+            .append(nettyTcnativeVersion)
+            .append(". If your application runs without issue this message can be ignored, otherwise please align the "
+                + "Netty versions you are using (this doesn't need to be the versions used by azure-core-http-netty if "
```

Comment:
Can you simplify the phrasing here?  You could reword it to say 

`Please align the Netty versions to be the same version (or higher)  used by azure-core-http-netty`

The phrasing should be simple and succinct (try using CoPilot) as it's going to be googled often.